src/databricks/labs/ucx/source_code/known.json

```diff
@@ -1690,6 +1690,276 @@
     "datasetsforecast.m5": [],
     "datasetsforecast.utils": []
   },
+  "dbacademy": {
```

Comment:
I can not find this package on Pypi" https://pypi.org/search/?q=dbacademy. Is it a relative package?

Reply:
no you install it using a git url

Reply:
Could you add the link? As a comment in the know.json?

Reply:
json does not support comments, the link is here https://github.com/databricks-academy/dbacademy/blob/main/README.md